ideas/311-desktop-team-proposal.md

```diff
@@ -0,0 +1,124 @@
+---
+id: 311-dekstop
+title: Desktop
+status: research/implementation
```

Comment:
research only probably for now - would be useful to sanity check with Rachel and Hester from Product and UX(R) POV